docs/Installing.asciidoc

```diff
@@ -299,6 +299,144 @@ For older versions of openQA, you can migrate from SQLite to PostgreSQL
 according to
 <<Pitfalls.asciidoc#db-migration,DB migration from SQLite to PostgreSQL>>
 
+==== Migrate PostgreSQL database on openSUSE
```

Comment:
This is now a section in the chapter "Basic configuration", better we move it to "Advanced".

Reply:
Ok, point taken. I'd only like to note that the steps are not optional anymore if we decide to use newer PostgreSQL features at some point (unless one doesn't care about the data).